R/dfeR-package.R

```diff
@@ -6,5 +6,5 @@
 ## usethis namespace: end
 
 #' @importFrom rlang .data
-#' @importFrom magrittr %>%
+#' @importFrom magrittr |>
```

Comment:
This line needs removing - `|>` is from base R so doesn't need importing and this line gives a warning when you run it locally as it tries to find it in magrittr

<img width="573" height="164" alt="Image" src="https://github.com/user-attachments/assets/04888025-15e9-4981-a3d9-10b2ed7a293d" />